app/controllers/QueuesController.scala

```diff
@@ -53,8 +53,7 @@ class QueuesController @Inject() (
           for {
             cases            <- casesService.getCasesByQueue(q, NoPagination(), types)
             queues           <- queuesService.getAll
-            caseCountByQueue <- casesService.countCasesByQueue(request.operator)
-          } yield Ok(views.html.queue(queues, q, caseCountByQueue, cases, types.mkString(",")))
+          } yield Ok(views.html.queue(queues, q, Map.empty, cases, types.mkString(",")))
```

Comment:
This controller is not used anywhere anymore but I can't delete it because removing all of the links to it is another (quite big) ticket